### PR TITLE
Markup and styles for new FeatureCard component.

### LINKED ISSUE #6722: Create markup and styles for the Campaign Card onboarding component.
image

^ this guy.

### PR DESCRIPTION
**Pull request by weerd**
#### What's this PR do?

This PR provides the full markup and styles for the `FeatureCard` component that showcases other campaigns a user can signup for during the onboarding process.

image
#### How should this be reviewed?

Checkout the component markup/JSX etc and the styles to see if it looks cool.
#### Any background context you want to provide?

I experienced a few CSS style conflicts while trying to work with creating the card within our established pattern library and approach to building new patterns. Something we should eventually discuss if this _card_ pattern is going to be the pattern for both the permalink card and the RB item gallery card in the future.
#### Relevant tickets

Fixes #6722 
#### Checklist
- [ ] Tested on staging.

---

@deadlybutter @DFurnes @sbsmith86 

cc @jessleenyc @angaither

### CODE REVIEW

**Comment by weerd** [comment]
Not sure why StyleCI is giving me crap on files I never touched 😕

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss [hunk 1]
```diff
@@ -32,11 +32,7 @@
 
 .slideshow__slide {
   .heading {
-    &.-beta {
-      color: #fff;
-      font-size: $font-medium;
-      font-weight: 600;
-    }
+    color: #fff;
```
Comment (sbsmith86) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss]: use `$white` here to?

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss]: good find!

**Comment by sbsmith86** [comment]
Looks good to me!

**Comment by weerd** [comment]
thanks @sbsmith86 💃

**Comment by itsjoekent** [comment]
react looks good 👍

**Pull request merged**